src/main/java/org/galatea/starter/utils/DateHelpers.java

```diff
@@ -0,0 +1,66 @@
+package org.galatea.starter.utils;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import org.galatea.starter.domain.IexHistoricalPriceDTO;
+
+public class DateHelpers {
+  public static List<LocalDate> getDates(String range) {
+    LocalDate end = LocalDate.now();
+    List<LocalDate> totalDates = new ArrayList<>();
+    LocalDate start = end.minusDays(30); // default
```

Comment:
This default range is duplicated in the IexService

Reply:
I updated it!